### PR TITLE
Bugfixing/controller

### PR DESCRIPTION
**Pull request by melvicaballero**
rename some packages and fix path ffmpeg of model convert audio and video.

### CODE REVIEW

File: src/main/java/com/jala/controller/ControllerConvertVideo.java [hunk 1]
```diff
@@ -73,7 +73,7 @@ public void actionPerformed(ActionEvent event) {
     }
 
     /**
-     * This method sends to ConvertVideo the criteria that was wrote by user.
+     * This method sends to ConvertVideo the criteria that was wrote by user
```
Comment (raulchoque-eng) [inline src/main/java/com/jala/controller/ControllerConvertVideo.java:76]: Put the period at end sentences.

File: src/main/java/com/jala/controller/ControllerConvertVideo.java [hunk 2]
```diff
@@ -99,7 +99,7 @@ private void sendPathToConvertAudio() {
      */
     private void saveCriteriaVideo() {
         String pathOrigin = viewConvert.getTxtPathFileOrigin();
-        String pathDestiny = viewConvert.getTxtFolderDestiny();
+        String pathDestination = viewConvert.getTxtFolderDestiny();
```
Comment (raulchoque-eng) [inline src/main/java/com/jala/controller/ControllerConvertVideo.java:102]: Use "pathOutput" how name or variable.

File: src/main/java/com/jala/model/convert/ConvertAudio.java [hunk 3]
```diff
@@ -55,15 +57,15 @@ public void convert(Object criteriaObject) {
...
+			log.error("Error in ConvertAudio", error);
 		}
 	}
 
 	/**
 	 * Convert the music with advances parameters like audioChannel, BitRate and SampleRate.
-	 * @param criteria
-	 * @param builder
-	 * @param fmpeg
+	 * @param criteria criteria for convert audio.
```
Comment (raulchoque-eng) [inline src/main/java/com/jala/model/convert/ConvertAudio.java]: Clear the "criteria" word for E.G. (@param criteria is for convert audio.)

File: src/main/java/com/jala/model/convert/ConvertAudio.java [hunk 4]
```diff
@@ -55,15 +57,15 @@ public void convert(Object criteriaObject) {
...
+	 * @param builder FFmpeg builder.
```
Comment (raulchoque-eng) [inline src/main/java/com/jala/model/convert/ConvertAudio.java:65]: Improve your comment.

File: src/main/java/com/jala/model/convert/ConvertAudio.java [hunk 5]
```diff
@@ -55,15 +57,15 @@ public void convert(Object criteriaObject) {
...
+	 * @param fmpeg converter app.
```
Comment (raulchoque-eng) [inline src/main/java/com/jala/model/convert/ConvertAudio.java:66]: improve your comment.

File: src/main/java/com/jala/model/search/asset/AssetText.java [hunk 6]
```diff
@@ -10,7 +10,7 @@
  * with Jala Foundation.
  */
 
-package com.jala.model.search.assetFile;
+package com.jala.model.search.asset;
 
 /**
  * AssetText class.
```
Comment (raulchoque-eng) [inline src/main/java/com/jala/model/search/asset/AssetText.java:16]: Put a description.

File: src/main/java/com/jala/controller/ControllerConvertVideo.java (hunk 1 above)
Comment (melvicaballero) [reply src/main/java/com/jala/controller/ControllerConvertVideo.java:76]: The end sentence is in the next line.

File: src/main/java/com/jala/controller/ControllerConvertVideo.java (hunk 2 above)
Comment (melvicaballero) [reply src/main/java/com/jala/controller/ControllerConvertVideo.java:102]: In the UI and Convert we will use pathDestination.

**Pull request merged**